framework/sandstone.cpp

```diff
@@ -102,6 +96,15 @@ struct HandleCloser
 using AutoClosingHandle = std::unique_ptr<void, HandleCloser>;
 #endif
 
+#include "sandstone_test_lists.h"
+static size_t builtin_test_list_sz;
+static struct test * const *builtin_test_list
+#if SANDSTONE_BUILTIN_TEST_LIST
+        ; // do nothing here
+#else
+        = { nullptr }; // statically initialize to empty list
+#endif
```

Comment:
Minor nit: these two variables don't need to be static globals. They're only used when processing the requested test list.